apps/aecore/src/aec_object_serialization.erl

```diff
@@ -78,6 +78,7 @@ tag(channel_slash_tx) -> 55;
 tag(channel_settle_tx) -> 56;
 tag(channel_offchain_tx) -> 57;
 tag(channel) -> 58;
+tag(trees_poi) -> 60;
```

Comment:
I understand the proof of inclusion is not part of consensus. May I suggest marking it with a comment in this file (e.g. `tag(trees_poi) -> 60 %% Not part of consensus.`)?

This also means no need to amend [protocol](https://github.com/aeternity/protocol/blob/fb7059ee91617718ce8c8356c18685db13ca64d6/serializations.md#table-of-object-tags).

Reply:
The proof of inclusion will be part of the channels' closing transactions so it will be used in the consesus. If a comment would be added - it will probably be short lived